controllers/component_controller.go

```diff
@@ -195,27 +194,7 @@ func (r *ComponentReconciler) UpdateValuesConfigmap(ctx context.Context, logger
 		versionStr := version.Version // https://golang.org/doc/faq#closures_and_goroutines
 		httpDonwloadURLs := version.URLs
 		g.Go(func() error {
```

Comment:
there is `g.SetLimit(workers)` in line 192, IMO, we should set this limit to `helm.ChartWorker` and no need to use `g.Go()`